src/style.css

```diff
@@ -0,0 +1,64 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-size: 16px;
+  padding-top: 20px;
+  margin-top: 0;
+  background-color: #fff;
+}
+
+.searchBar {
+  display: block;
+}
+
+.searchInput {
+  width: 100%;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 1px 1px 8px rgba(61, 78, 97, 0.1);
+  font-family: Avenir, sans-serif;
```

Comment:
set font-family in `body` or `html` (for the entire page) and then set `inherit` here to overwrite the default browser value for input